bin/dev_hdd1/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
Are you sure you can delete this? I doesn't say "Note: This folder has to exist. Once the User Account manager is implemented, make sure it creates this folder in case it's missing and delete this .gitignore file. " as previous ones.

Reply:
yes, this instruction is very old, and so neko have codded one thing little old, so now that is obsolete and useless